main.tf

```diff
@@ -1,5 +1,7 @@
 #
+
```

Comment:
This is my biggest complaint with the `terraform fmt` subcommand. I sometimes try to remove the extra space it introduces, but not a big deal.